sucury.py

```diff
@@ -242,16 +306,18 @@ def draw_grid():
             elif event.key == pygame.K_LEFT:  # Left arrow:  move left
                 snake.ymov = 0
                 snake.xmov = -1
-            elif event.key == pygame.K_q:     # Q         : quit game
+            elif event.key == pygame.K_q:     # Q:           quit game
                 pygame.quit()
                 sys.exit()
-            elif event.key == pygame.K_p:     # S         : pause game
+            elif event.key == pygame.K_p:     # S:           pause game
+                game_on = not game_on
+            elif event.key == pygame.K_c:     # C:           show color menu
+                show_color_menu = not show_color_menu
                 game_on = not game_on
```

Comment:
Incomplete pause logic. It's possible to unpause while the menu is open and open the menu with the game paused. Both of those are bugs.